etc/profile-m-z/ssh.profile

```diff
@@ -20,6 +20,7 @@ include disable-programs.inc
 
 whitelist ${RUNUSER}/gnupg/S.gpg-agent.ssh
 whitelist ${RUNUSER}/keyring/ssh
+whitelist ${RUNUSER}/gcr/ssh
```

Comment:
```suggestion
whitelist ${RUNUSER}/gcr/ssh
whitelist ${RUNUSER}/gnupg/S.gpg-agent.ssh
whitelist ${RUNUSER}/keyring/ssh
```

Sort.

Reply:
Thanks for this, I ran the sort.py script from `contrib/` but I didn't do anything for some reason.

Reply:
> Thanks for this, I ran the sort.py script from `contrib/` but I didn't do
> anything for some reason.

No problem; that script only sorts comma-separated items within a given line
for specific commands.  The lines themselves are currently sorted manually.